tests/datavolume_test.go

```diff
@@ -763,18 +790,19 @@ var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]DataVolume tests",
 	Describe("Create/Delete same datavolume in a loop", func() {
 		Context("retry loop", func() {
 			dataVolumeName := "dv1"
-			url := fmt.Sprintf(utils.TinyCoreIsoURL, f.CdiInstallNs)
 			numTries := 5
 			for i := 1; i <= numTries; i++ {
 				It(fmt.Sprintf("[test_id:3939][test_id:3940][test_id:3941][test_id:3942][test_id:3943]should succeed on loop %d", i), func() {
+					url := fmt.Sprintf(utils.TinyCoreIsoURL, f.CdiInstallNs)
 					dataVolume := utils.NewDataVolumeWithHTTPImport(dataVolumeName, "1Gi", url)
 
 					By(fmt.Sprintf("creating new datavolume %s", dataVolume.Name))
 					dataVolume, err := utils.CreateDataVolumeFromDefinition(f.CdiClient, f.Namespace.Name, dataVolume)
 					Expect(err).ToNot(HaveOccurred())
 
 					By(fmt.Sprintf("waiting for datavolume to match phase %s", cdiv1.Succeeded))
-					utils.WaitForDataVolumePhase(f.CdiClient, f.Namespace.Name, cdiv1.Succeeded, dataVolume.Name)
+					err = utils.WaitForDataVolumePhase(f.CdiClient, f.Namespace.Name, cdiv1.Succeeded, dataVolume.Name)
+					Expect(err).ToNot(HaveOccurred())
```

Comment:
good catch, 
I think I know why it happens, some of our test utils do check and fail inside the function logic, and some are just returning an err value so the caller can assert it,

Reply:
Yeah, well I noticed this should have failed while I was testing and the test 'passed' so now it should fail properly if it fails.